src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -1885,7 +1885,7 @@ protected bool TryGetInterproceduralAnalysisResult(IOperation operation, out TAn
                 copyAnalysisResultOpt?.ControlFlowGraph ??
                 valueContentAnalysisResultOpt?.ControlFlowGraph ??
                 getCfg();
-            if (cfg == null)
+            if (cfg == null || !cfg.SupportsFlowAnalysis())
```

Comment:
@dotpaul - The only real fix in the PR is the change here and the null check below. All the others are just symbol renames. I am planning to merge this in when tests pass to ensure this change is included in https://github.com/dotnet/roslyn/pull/36097

Reply:
LGTM